api/src/main/java/org/pmiops/workbench/profile/ProfileService.java

```diff
@@ -135,4 +168,155 @@ public void validateInstitutionalAffiliation(Profile profile) {
       throw new BadRequestException(msg);
     }
   }
+
+  public void updateProfileForUser(DbUser user, Profile updatedProfile, Profile previousProfile) {
```

Comment:
All this updateXForUser should be unchanged from what it was in the ProfileController.